src/contants.js

```diff
@@ -268,28 +333,160 @@ export const employeeImageMapper = (employeeId) => {
       return id22;
     case 23:
       return id23;
+    case 24:
+      return id24;
+    case 25:
+      return id25;
+    case 26:
+      return id26;
+    case 27:
+      return id27;
+    case 28:
+      return id28;
```

Comment:
Is there any better way to do this?

Reply:
I will look into whether there's a better way to do this.